src/Leanplum.js

```diff
@@ -383,10 +383,22 @@ export default class Leanplum {
         )
   }
 
-  static trackMessage(event, messageId) {
+  // TODO: need probably to add the event (aka exit, accept, cancel)
+  //  to build valid filter value: '.c23423938 exit'
+  //  either VarCache getMessageById or pass message or pass message params
+  static trackMessage( event, messageId) {
```

Comment:
I don't see this whole function to be present on native SDKs and I can see only confusion when adding another name to already existent `track` and `advanceTo`. Leanplum knows only events and states (and so does business), so no need for new function from my point of view.